components/web/apache2-modules/mod_perl/apache-perl.p5m

```diff
@@ -289,95 +289,95 @@ file path=usr/apache2/2.4/lib/perl/auto/mod_perl2/.packlist
 file path=usr/apache2/2.4/lib/perl/mod_perl2.pm
```

Comment:
I'm afraid these files are hardly usable here:
```
$ perl -e 'use mod_perl2'
Can't locate mod_perl2.pm in @INC (you may need to install the mod_perl2 module) (@INC contains: /usr/perl5/site_perl/5.36/i86pc-solaris-thread-multi-64 /usr/perl5/site_perl/5.36 /usr/perl5/vendor_perl/5.36/i86pc-solaris-thread-multi-64 /usr/perl5/vendor_perl/5.36 /usr/perl5/5.36/lib/i86pc-solaris-thread-multi-64 /usr/perl5/5.36/lib) at -e line 1.
BEGIN failed--compilation aborted at -e line 1.
$
```
Would it be possible to move them to `/usr/perl5/vendor_perl/5.36/`?

Reply:
Please look at https://metacpan.org/dist/mod_perl/view/docs/api/APR.pod.  It is expected that `perl -MAPR::UUID -le 'print APR::UUID->new->format'` works.  But, unfortunately, it does not:
```
$ perl -MAPR::UUID -le 'print APR::UUID->new->format'
Can't locate APR/UUID.pm in @INC (you may need to install the APR::UUID module) (@INC contains: /usr/perl5/site_perl/5.36/i86pc-solaris-thread-multi-64 /usr/perl5/site_perl/5.36 /usr/perl5/vendor_perl/5.36/i86pc-solaris-thread-multi-64 /usr/perl5/vendor_perl/5.36 /usr/perl5/5.36/lib/i86pc-solaris-thread-multi-64 /usr/perl5/5.36/lib).
BEGIN failed--compilation aborted.
$
```
And it won't become better with this PR.